### ISSUE TITLE
Debian Buster installation fails

### ISSUE THREAD

**Issue by ckorder** [tmiland/Invidious-Updater#35]
the bug:
invidious.service: Failed with result 'exit-code'.
ERROR:  relation "nonces" does not exist
LINE 1: DELETE FROM nonces * WHERE expire < current_timestamp;
                    ^
➜ Truncating videos table.
ERROR:  relation "videos" does not exist
Dec 11 16:11 xyz634 systemd[1]: Started Invidious (An alternative YouTube front-end).
Dec 11 16:1 xyz634 invidious[28093]: Unhandled exception: mapping values are not allowed in this context at line 2, column 15 (YAML::ParseException)

With Docker everything works well.

// invidious_update.sh: line 1293: ./invidious_update.sh: Permission denied
// is not causing the bug, i guess

Reproduce:

sadly i did not safe selection 1, but it was like 2 not showing directly that it does not work.
only the potential hint: Shard "lsquic" version (2.18.1) doesn't match tag version (2.18.1-1)

```

Select an option [1-10]: 2

Invidious is ready to be updated, press any key to continue...a
➜ Pulling Invidious from GitHub
fatal: Unable to mark file config/config.yml
Already up to date.
Saved working directory and index state WIP on master: b566c4b Untrack config.yml (#1557)
Branch 'master' set up to track remote branch 'master' from 'origin'.
Reset branch 'master'
Your branch is up to date with 'origin/master'.

-- Rebuilding /home/invidious/invidious
Resolving dependencies
Fetching https://github.com/will/crystal-pg.git
Fetching https://github.com/crystal-lang/crystal-sqlite3.git
Fetching https://github.com/kemalcr/kemal.git
Fetching https://github.com/ysbaddaden/pool.git
Fetching https://github.com/omarroth/protodec.git
Fetching https://github.com/iv-org/lsquic.cr.git
Fetching https://github.com/crystal-lang/crystal-db.git
Fetching https://github.com/luislavena/radix.git
Fetching https://github.com/jeromegn/kilt.git
Fetching https://github.com/crystal-loot/exception_page.git
Shard "lsquic" version (2.18.1) doesn't match tag version (2.18.1-1)
Using db (0.9.0)
Using pg (0.21.1)
Using sqlite3 (0.16.0)
Using radix (0.3.9)
Using kilt (0.4.0)
Using exception_page (0.1.4)
Using kemal (0.26.1 at dfe7dca)
Using pool (0.2.3)
Using protodec (0.1.3)
Using lsquic (2.18.1-1)
Writing shard.lock
Resolving dependencies
Fetching https://github.com/will/crystal-pg.git
Fetching https://github.com/crystal-lang/crystal-db.git
Fetching https://github.com/crystal-lang/crystal-sqlite3.git
Fetching https://github.com/kemalcr/kemal.git
Fetching https://github.com/luislavena/radix.git
Fetching https://github.com/jeromegn/kilt.git
Fetching https://github.com/crystal-loot/exception_page.git
Fetching https://github.com/ysbaddaden/pool.git
Fetching https://github.com/omarroth/protodec.git
Fetching https://github.com/iv-org/lsquic.cr.git
Shard "lsquic" version (2.18.1) doesn't match tag version (2.18.1-1)
Using db (0.9.0)
Using pg (0.21.1)
Using sqlite3 (0.16.0)
Using radix (0.3.9)
Using kilt (0.4.0)
Using exception_page (0.1.4)
Using kemal (0.26.1 at dfe7dca)
Using pool (0.2.3)
Using protodec (0.1.3)
Using lsquic (2.18.1-1)
/home/invidious/invidious

✔ Done Rebuilding /home/invidious/invidious 

-- restarting Invidious
● invidious.service - Invidious (An alternative YouTube front-end)
   Loaded: loaded (/lib/systemd/system/invidious.service; enabled; vendor preset: enabled)
   Active: activating (auto-restart) (Result: exit-code) since Fri 2020-12-11 16:49:33 MSK; 1s ago
  Process: 27727 ExecStart=/home/invidious/invidious/invidious -b -p -o invidious.log (code=exited, status=1/FAILURE)
 Main PID: 27727 (code=exited, status=1/FAILURE)

Dec 11 16:49:33 xyz634 systemd[1]: invidious.service: Failed with result 'exit-code'.

✔ Invidious has been restarted 
invidious_update.sh: line 1293: ./invidious_update.sh: Permission denied


##############################################################################

# Result in:  502 Bad Gateway nginx (of course i added the vhost with correct ssl cert)
# not like the default 502 Bad Gateway like after reboot. it is permanent

##############################################################################

Select an option [1-10]: 5
Are you sure you want to run Database Maintenance? Enter [y/n]: y

Your Invidious database name: 

Is that correct? Enter [y/n]: y
✗ stopping Invidious...
➜ Running Maintenance on  
➜ Deleting expired tokens
ERROR:  relation "nonces" does not exist
LINE 1: DELETE FROM nonces * WHERE expire < current_timestamp;
                    ^
➜ Truncating videos table.
ERROR:  relation "videos" does not exist
➜ Vacuuming .
vacuumdb: vacuuming database "postgres"
vacuumdb: query failed: ERROR:  relation "videos" does not exist
LINE 4:   AND c.oid OPERATOR(pg_catalog.=) 'videos'::pg_catalog.regc...
                                           ^
vacuumdb: query was: SELECT c.relname, ns.nspname
 FROM pg_catalog.pg_class c, pg_catalog.pg_namespace ns
 WHERE c.relnamespace OPERATOR(pg_catalog.=) ns.oid
  AND c.oid OPERATOR(pg_catalog.=) 'videos'::pg_catalog.regclass;
➜ Reindexing .
✔ Maintenance on  done.
➜ Restarting postgresql...
✔ Restarting postgresql done.
● postgresql.service - PostgreSQL RDBMS
   Loaded: loaded (/lib/systemd/system/postgresql.service; enabled; vendor preset: enabled)
   Active: active (exited) since Fri 2020-12-11 16:11 MSK; 52ms ago
  Process: 28086 ExecStart=/bin/true (code=exited, status=0/SUCCESS)
 Main PID: 28086 (code=exited, status=0/SUCCESS)

Dec 11 16:11 xyz634 systemd[1]: Starting PostgreSQL RDBMS...
Dec 11 16:11 xyz634 systemd[1]: Started PostgreSQL RDBMS.
➜ Restarting Invidious...
✔ Restarting Invidious done.
● invidious.service - Invidious (An alternative YouTube front-end)
   Loaded: loaded (/lib/systemd/system/invidious.service; enabled; vendor preset: enabled)
   Active: active (running) since Fri 2020-12-11 16:11 MSK; 30ms ago
 Main PID: 28093 (invidious)
    Tasks: 4 (limit: 2356)
   Memory: 2.2M
   CGroup: /system.slice/invidious.service
           └─28093 /home/invidious/invidious/invidious -b -p -o invidious.log

Dec 11 16:11 xyz634 systemd[1]: Started Invidious (An alternative YouTube front-end).
Dec 11 16:11 xyz634 invidious[28093]: Unhandled exception: mapping values are not allowed in this context at line 2, column 15 (YAML::ParseException)
```

**Comment by tmiland** [comment]
Hello, did you enter your root password?

**Comment by tmiland** [comment]
Did a successful test-install on Debian 10 (Buster) just now:

<details><summary>*truncated* **click to view**</summary><p>


```

 ╔═══════════════════════════════════════════════════════════════════╗
 ║                        Invidious Update.sh                        ║
 ║               Automatic update script for Invidious               ║
 ║                      Maintained by @tmiland                       ║
 ║                          version: 1.4.9                           ║
 ╚═══════════════════════════════════════════════════════════════════╝

Thank you for using the Invidious Update.sh script.



Documentation for this script is available here: 
 ➜ https://github.com/tmiland/Invidious-Updater


Let's go through some configuration options.

Do you want to enter advanced options? [y/n]: n


You entered: 

 ✔ branch        : master
 ✔ domain        : invidious.tube
 ✔ ip adress     : localhost
 ✔ port          : 3000
 ✔ external port : 
 ✔ dbname        : invidious
 ✔ dbpass        : kemal
 ✔ https only    : false
 


Invidious is ready to be installed, press any key to continue...

➜ Downloading Invidious from GitHub
Cloning into 'invidious'...
remote: Enumerating objects: 35, done.
remote: Counting objects: 100% (35/35), done.
remote: Compressing objects: 100% (25/25), done.
remote: Total 13514 (delta 15), reused 24 (delta 10), pack-reused 13479
Receiving objects: 100% (13514/13514), 18.20 MiB | 7.97 MiB/s, done.
Resolving deltas: 100% (8639/8639), done.
/bin/cp: cannot stat '/home/invidious/invidious/config/config.yml': No such file or directory
fatal: Unable to mark file config/config.yml
Branch 'master' set up to track remote branch 'master' from 'origin'.
Reset branch 'master'
Your branch is up to date with 'origin/master'.
➜ Done
/home/invidious
Synchronizing state of postgresql.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install enable postgresql
➜ Creating user kemal with password kemal 
CREATE ROLE
➜ Creating database invidious with owner kemal
CREATE DATABASE
➜ Grant all on database invidious to user kemal
GRANT
➜ Running /home/invidious/invidious/config/sql/annotations.sql 
CREATE TABLE
GRANT
➜ Running /home/invidious/invidious/config/sql/channels.sql 
CREATE TABLE
GRANT
CREATE INDEX
➜ Running /home/invidious/invidious/config/sql/channel_videos.sql 
CREATE TABLE
GRANT
CREATE INDEX
➜ Running /home/invidious/invidious/config/sql/nonces.sql 
CREATE TABLE
GRANT
CREATE INDEX
➜ Running /home/invidious/invidious/config/sql/playlists.sql 
CREATE TYPE
CREATE TABLE
GRANT
➜ Running /home/invidious/invidious/config/sql/playlist_videos.sql 
CREATE TABLE
GRANT
➜ Running /home/invidious/invidious/config/sql/session_ids.sql 
CREATE TABLE
GRANT
CREATE INDEX
➜ Running /home/invidious/invidious/config/sql/users.sql 
CREATE TABLE
GRANT
CREATE INDEX
➜ Running /home/invidious/invidious/config/sql/videos.sql 
CREATE TABLE
GRANT
CREATE INDEX
✔ Finished Database section
channel_threads: 1
  feed_threads: 1
  db:
    user: kemal
    password: kemal
    host: localhost
    port: 5432
    dbname: invidious
  full_refresh: false
  https_only: false
  domain:
➜ Updating config.yml with new info...
✔ Done.
Resolving dependencies
Fetching https://github.com/will/crystal-pg.git
Fetching https://github.com/crystal-lang/crystal-sqlite3.git
Fetching https://github.com/kemalcr/kemal.git
Fetching https://github.com/ysbaddaden/pool.git
Fetching https://github.com/omarroth/protodec.git
Fetching https://github.com/iv-org/lsquic.cr.git
Fetching https://github.com/crystal-lang/crystal-db.git
Fetching https://github.com/luislavena/radix.git
Fetching https://github.com/jeromegn/kilt.git
Fetching https://github.com/crystal-loot/exception_page.git
Shard "lsquic" version (2.18.1) doesn't match tag version (2.18.1-1)
Installing db (0.9.0)
Installing pg (0.21.1)
Installing sqlite3 (0.16.0)
Installing radix (0.3.9)
Installing kilt (0.4.0)
Installing exception_page (0.1.4)
Installing kemal (0.26.1 at dfe7dca)
Installing pool (0.2.3)
Installing protodec (0.1.3)
Installing lsquic (2.18.1-1)
Writing shard.lock
Resolving dependencies
Fetching https://github.com/will/crystal-pg.git
Fetching https://github.com/crystal-lang/crystal-db.git
Fetching https://github.com/crystal-lang/crystal-sqlite3.git
Fetching https://github.com/kemalcr/kemal.git
Fetching https://github.com/luislavena/radix.git
Fetching https://github.com/jeromegn/kilt.git
Fetching https://github.com/crystal-loot/exception_page.git
Fetching https://github.com/ysbaddaden/pool.git
Fetching https://github.com/omarroth/protodec.git
Fetching https://github.com/iv-org/lsquic.cr.git
Shard "lsquic" version (2.18.1) doesn't match tag version (2.18.1-1)
Using db (0.9.0)
Using pg (0.21.1)
Using sqlite3 (0.16.0)
Using radix (0.3.9)
Using kilt (0.4.0)
Using exception_page (0.1.4)
Using kemal (0.26.1 at dfe7dca)
Using pool (0.2.3)
Using protodec (0.1.3)
Using lsquic (2.18.1-1)
✔ Invidious service has been successfully installed!
● invidious.service - Invidious (An alternative YouTube front-end)
   Loaded: loaded (/lib/systemd/system/invidious.service; enabled; vendor preset: enabled)
   Active: active (running) since Sat 2020-12-12 20:05:47 CET; 138ms ago
 Main PID: 4631 ((nvidious))
    Tasks: 0 (limit: 2338)
   Memory: 0B
   CGroup: /system.slice/invidious.service
           └─4631 (nvidious)

Dec 12 20:05:47 debian10-uni systemd[1]: Started Invidious (An alternative …nd).
Hint: Some lines were ellipsized, use -l to show in full.
Adding logrotate configuration...
/home/invidious/invidious/invidious.log {
    rotate 4
    weekly
    notifempty
    missingok
    compress
    minsize 1048576
}
 (done)


 ╔═══════════════════════════════════════════════════════════════════╗
 ║                        Invidious Update.sh                        ║
 ║               Automatic update script for Invidious               ║
 ║                      Maintained by @tmiland                       ║
 ║                          version: 1.4.9                           ║
 ╚═══════════════════════════════════════════════════════════════════╝




Thank you for using the Invidious Update.sh script.



✔ Invidious install done. Now visit http://localhost:3000




Documentation for this script is available here: 
 ➜ https://github.com/tmiland/Invidious-Updater



 ╔═══════════════════════════════════════════════════════════════════╗
 ║                        Invidious Update.sh                        ║
 ║               Automatic update script for Invidious               ║
 ║                      Maintained by @tmiland                       ║
 ║                          version: 1.4.9                           ║
 ╚═══════════════════════════════════════════════════════════════════╝

Welcome to the Invidious Update.sh script.

What do you want to do?

  1) Install Invidious          6) Start, Stop or Restart   
  2) Update Invidious           7) Uninstall Invidious      
  3) Deploy with Docker         8) Set up PostgreSQL Backup 
  4) Add Swap Space             9) Install Nginx            
  5) Run Database Maintenance  10) Exit                     
 

Documentation for this script is available here: 
 ➜ https://github.com/tmiland/Invidious-Updater

Select an option [1-10]: 
 ```
</p></details>

**Comment by tmiland** [comment]
Fixed in new release: [v1.5.0](https://github.com/tmiland/Invidious-Updater/releases/tag/v1.5.0)

**Comment by ckorder** [comment]
> Hello, did you enter your root password?
yes but im running as sudo with disabled root user /sbin/nologin as shell
>Edit* Are you on the latest version of the script?
yes.
`Hello,

change `port: 3000` in `config.yml` for now. 

I'll have a look later. :+1:

_Originally posted by @tmiland in https://github.com/tmiland/Invidious-Updater/issues/36#issuecomment-743799707_`
okay i will try it within next 2 weeks. (very busy right now) 
and check the fix as well. thank you very much for the quick response!
